src/lib.rs

```diff
@@ -430,6 +433,13 @@ pub use crate::vector::CxxVector;
 pub use crate::weak_ptr::WeakPtr;
 pub use cxxbridge_macro::bridge;
 
+/// Synonym for `CxxChar`.
+///
+/// To avoid confusion with Rust's built-in char type you probably
+/// shouldn't import this type with `use`. Instead, write `cxx::Char`, or
+/// import and use `CxxChar`.
+pub type Char = CxxChar;
```

Comment:
I wasn't entirely sure of the reason for these aliases, so just following the existing pattern. 🤷‍♂️